app/views/reports/setup.js.erb

```diff
@@ -21,7 +21,6 @@ $('#defined_reports_step_2').html("<%= raw escape_javascript(render(:partial =>
 $('#report_selection').hide();
 $('#defined_reports, #defined_reports_step_2').show();
 // $('#defined_reports_step_1').dialog('close');
-$("select").selectpicker()
```

Comment:
@wtholt Because disabled the existing functionality isn't a fix. My commit fixes the bug, and keeps the functionality of selectpickers for reports that used it (at this point, only the survey report).